profile/README.md

```diff
@@ -0,0 +1,90 @@
+# Rust Bitcoin Community
+
+A Series of projects to implement various Bitcoin protocols in Rust.
+
```

Comment:
```suggestion

## TLDR

`rust-bitcoin` hosts the [`bitcoin`](https://crates.io/crates/bitcoin) crate that
supports the Bitcoin network protocol and associated primitives.
It is designed for Rust programs built to work with the Bitcoin network.

```

Something like this. Feel free to edit or add more to it.

The idea is that if a newcomer sees the profile README he gets a nice "elevator pitch" of what `rust-bitcoin` is all about without having "death by analysis" by having to go into a bunch of different crates' READMEs.

I would have much appreciated something like this when I was starting to play around with `rust-bitcoin` a while ago.

Feel free to ignore this if you find that my arguments are invalid.

Reply:
Good suggestion, I'll work this in.

Reply:
Used as is (with column width 100).